packages/salesforcedx-vscode-core/src/orgBrowser/metadataType.ts

```diff
@@ -111,6 +112,13 @@ export class TypeUtils {
   }
 
   public getFolderForType(metadataType: string): string {
-    return `${metadataType === 'EmailTemplate' ? 'Email' : metadataType}Folder`;
+    switch (metadataType) {
+      case 'CustomObject':
+        return metadataType;
+      case 'EmailTemplate':
+        return 'EmailFolder';
```

Comment:
The production version had the logic of, "if metadataType is EmailTemplate, then return 'Email'.  This was a bug and should have returned 'EmailFolder'.